.github/workflows/testpydra.yml

```
@@ -16,7 +16,7 @@ jobs:
     strategy:
       matrix:
         os: [macos-latest, ubuntu-latest, windows-latest]
-        python-version: [3.7, 3.8, 3.9, "3.10"]
+        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
```

Windows is waiting on https://github.com/lxml/lxml/pull/356. Maybe we should exclude that from the matrix for now and create a PR to re-include it to remind us to check periodically?

```suggestion
        python-version: ['3.7', '3.8', '3.9', '3.10', '3.11']
        exclude:
          - os: 'windows-latest'
            python-version: '3.11'
```

---

It's actually [merged](https://github.com/lxml/lxml/pull/360) and scheduled for release in version `4.9.2`. People are pushing for it but no idea as to when exactly.

---

TIL, I did not know about this `exclude` option. It worked beautifully.